android-components/components/feature/customtabs/src/main/java/mozilla/components/feature/customtabs/CustomTabWindowFeature.kt

```diff
@@ -87,13 +85,12 @@ class CustomTabWindowFeature(
                         val uri = windowRequest.url.toUri()
                         // This could only fail if the above intent is for our application
                         // and we are not registered to handle its schemes.
-                        // Let's log this to better asses how often this happens in real world and
-                        // if we need to add new schemes to properly support this workflow.
-                        // See Fenix #8412
                         try {
                             intent.launchUrl(activity, uri)
                         } catch (e: ActivityNotFoundException) {
-                            onLaunchUrlFallback(uri)
+                            // Workaround for unsupported schemes
+                            // See https://bugzilla.mozilla.org/show_bug.cgi?id=1878704
+                            state.engineState.engineSession?.loadUrl(windowRequest.url)
```

Comment:
since this fix is in AC, will this need a change log addition?

Reply:
Good idea.  I'll add this to changelog before landing.  Thanks